Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OpenKAT/nl-kat-coordination](https://hosted.weblate.org/projects/openkat/nl-kat-coordination/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/openkat/nl-kat-coordination/horizontal-auto.svg)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/minvws/nl-kat-coordination?pullRequest=3870) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/minvws/nl-kat-coordination?pullRequest=3870) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/minvws/nl-kat-coordination?pullRequest=3870) it.</sub>

weblate committed:
Translated using Weblate

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/openkat/nl-kat-coordination/
Translation: OpenKAT/nl-kat-coordination

weblate committed:
Translated using Weblate

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/openkat/nl-kat-coordination/
Translation: OpenKAT/nl-kat-coordination

weblate committed:
Translated using Weblate

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/openkat/nl-kat-coordination/
Translation: OpenKAT/nl-kat-coordination

weblate committed:
Translated using Weblate

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Wim Benes <fryskefirefox@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openkat/nl-kat-coordination/
Translate-URL: https://hosted.weblate.org/projects/openkat/nl-kat-coordination/fy/
Translation: OpenKAT/nl-kat-coordination

weblate committed:
Translated using Weblate

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Wim Benes <fryskefirefox@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/openkat/nl-kat-coordination/
Translate-URL: https://hosted.weblate.org/projects/openkat/nl-kat-coordination/nl/
Translation: OpenKAT/nl-kat-coordination

weblate committed:
Translated using Weblate

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/openkat/nl-kat-coordination/
Translation: OpenKAT/nl-kat-coordination

weblate committed:
Translated using Weblate

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/openkat/nl-kat-coordination/
Translation: OpenKAT/nl-kat-coordination

underdarknl committed:
Merge branch 'main' into weblate-openkat-nl-kat-coordination
